inlineplz/linters/jshint.py

```diff
@@ -3,10 +3,37 @@
 from __future__ import unicode_literals
 
 import xmltodict
+import os.path
```

Comment:
[inline-plz]: `prospector: pylint: standard import "import os.path" should be placed before "import xmltodict" (wrong-import-order)`